llvm/test/CodeGen/SPIRV/llvm-intrinsics/is_fpclass.ll

```diff
@@ -0,0 +1,367 @@
+; RUN: llc -verify-machineinstrs -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
```

Comment:
Please also have a test with more-than-one masks used at once. I suspect there might be redundant/duplicating OpBitwiseAnd instructions generated.

Reply:
Good idea, thanks. I've added `@isfpclass_1_isnan_or_normal`. Turns out there actually aren't redundant `OpBitwiseAnd` being generated.

The worst thing I've seen is the redundant `OpLogicalOr %[[#BoolTy]] %[[#False]] %[[#T2]]` everywhere. Those would ideally be optimized out, perhaps either at the point of creation or after ISel. I thought I'd leave that for future work.